src/main/java/com/notably/logic/commands/OpenCommand.java

```diff
@@ -0,0 +1,21 @@
+package com.notably.logic.commands;
+
+import com.notably.commons.core.path.Path;
+import com.notably.logic.commands.exceptions.CommandException;
+import com.notably.model.BlockManager;
+
+/**
+ * Represents a command that opens a path.
+ */
+public class OpenCommand extends Command {
+    public static final String COMMAND_WORD = "open";
+
+    private final Path path;
```

Comment:
Let's make the whitespaces and grouping more consistent! To do that, let's group the fields together, and separate them from the constructor with a whitespace

Reply:
Noted!